src/web/app/pages-admin/admin-search-page/admin-search-page.component.spec.ts

```diff
@@ -471,8 +531,8 @@ describe('AdminSearchPageComponent', () => {
       newRegistrationKey: 'newKey',
     }));
 
-    const spyStatusMessageService: any = spyOn(statusMessageService, 'showSuccessToast').and.callFake(
-        (args: string): void => {
+    const spyStatusMessageService: any = spyOn(statusMessageService, 'showSuccessToast')
+        .and.callFake((args: string) => {
```

Comment:
Still seems like a bunch of unnecessary changes in this file

Reply:
The re-indentation was done because it exceeds the line limit, I double-checked my indentation settings in my IDE and seems to be correct also (I'm following what is specified in https://github.com/TEAMMATES/teammates/blob/master/docs/ide-setup.md).